package.json

```diff
@@ -12,7 +12,7 @@
     "build": "node tools/build.js",
     "build-docs": "npm run bootstrap-docs && npm -C www run build",
     "lint": "eslint www/src src test tools *.js",
-    "release": "release",
+    "release": "npm run build-docs && release",
```

Comment:
isn't this already part of `build`? i thought we already did all this

Reply:
I think I removed it along the way bc it was weird to build do a with the library? Idk if this is nicer